platforms/arm_atsam/bootloaders/md_boot.c

```diff
@@ -63,3 +63,7 @@ void bootloader_jump(void) {
     while (1)
         ; // Wait on timeout
 }
+
+__attribute__((weak)) void mcu_reset(void) {
+    // not certain how to setup wdt here
+}
```

Comment:
Kinda minor, but you cant use watchdog, as its use will trigger the device to stay in the bootloader
```suggestion
__attribute__((weak)) void mcu_reset(void) {}
```

Reply:
Will see if `NVIC_SystemReset()` works.

Reply:
Works fine.
```suggestion
__attribute__((weak)) void mcu_reset(void) {
    NVIC_SystemReset();
}
```